fapolicy_analyzer/ui/main_window.py

```diff
@@ -358,11 +362,49 @@ def on_saveAsMenu_activate(self, menuitem, data=None):
         fcd.destroy()
 
     def on_aboutMenu_activate(self, *args):
-        aboutDialog = self.get_object("aboutDialog")
-        aboutDialog.set_transient_for(self.window)
-        aboutDialog.set_version(f"v{app_version}")
-        aboutDialog.run()
-        aboutDialog.hide()
+        rpm_path = ""
+        try:
+            rpm_path = subprocess.getoutput(
```

Comment:
The json file can be marked as a resource and can be looked up like a glade file, we do not have to look it up with a call to rpm.

The build-info file can be wired into the setup.py like the glade files are, and because there is a placeholder build-info file, the setup.py build will work from both a dev-env and rpm install.